pkg/util/node.go

```diff
@@ -49,3 +56,28 @@ func IsNodeAddressTypeSupported(addrType corev1.NodeAddressType) bool {
 	}
 	return false
 }
+
+func GetReservedByNodeAnno(anno map[string]string) corev1.ResourceList {
+	rl := make(corev1.ResourceList)
+	reserved, ok := anno[apiext.AnnotationNodeReservation]
+	if !ok {
```

Comment:
If there is no AnnotationNodeReservation, just return nil.

Reply:
As a basic help function, it is better not to swallow internal errors, and it would be better to let the upper-level module decide how to deal with it.